assembl/views/api2/discussion.py

```diff
@@ -810,16 +841,20 @@ def users_csv_export(request):
                     users[user.id][key] += ', ' + value
 
     for post in posts:
+        if not post.id:
+            continue
```

Comment:
Huh? In what scenario will there be posts in the db that do not have an id?

Reply:
this happens in test.. dnw